# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/75_mm-2.7-remove_tilde.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/bin/sun-message.csh~ mm2.7/src/bin/sun-message.csh~
--- mm2.7.orig/src/bin/sun-message.csh~	2003-05-14 23:58:27.000000000 +0200
+++ mm2.7/src/bin/sun-message.csh~	1970-01-01 01:00:00.000000000 +0100
@@ -1,63 +0,0 @@
-#!/bin/csh -fb
-# (The "-fb" might need to be changed to "-f" on some systems)
-#
-#
-# This is sun-message.  It looks at $2 to figure out how to decode $1, then gives the
-# user a short menu of choices, display, save, or quit.
-#
-mkdir /tmp/decode.$$
-cd /tmp/decode.$$
-
-if (! $?METAMAIL_PAGER) then
-    set METAMAIL_PAGER=more
-endif
-
-if ($2 == "uuencode") then
-    uudecode $1
-
-    echo "The following file was uudecoded:"
-    echo ""
-
-    set defans = "1"
-    while (1)
-	ls -l
-	set fn = *
-
-	echo ""
-	echo "Please choose one:"
-	echo ""
-	echo "1 -- Display it as ASCII text"
-	echo "2 -- Save it as a file"
-	echo "3 -- Quit this menu"
-	echo ""
-	echo -n "Which do you prefer (1 - 3)? [$defans] "
-
-	set ans = $< 
-	if (x$ans == x) then
-	    set ans = $defans
-	endif
-	if ($ans == 3)  then
-	    rm $1
-	    cd /tmp; /bin/rm -rf /tmp/decode.$$
-	    exit 0
-	else if ($ans == 1) then
-	    ${METAMAIL_PAGER} $fn
-	else if ($ans == 2) then
-	    set nfn = ""
-	    echo -n "Save as: $HOME/"
-	    set nfn = $<
-	    if (x$nfn != x) then
-		/bin/cp $fn ${HOME}/$nfn
-	    else
-		echo "Not Saved."
-	    endif
-	else
-	    echo "Invalid choice."
-	endif
-	set defans = "3"
-    end
-else
-    ${METAMAIL_PAGER} $1
-endif
-
-
